dinstaller-cli/src/main.rs

```diff
@@ -64,7 +64,18 @@ async fn show_progress(client: &ManagerClient<'_>) {
     let mut progress = client.progress().await.unwrap();
     eprintln!("Showing progress with max steps {:?}", progress.max_steps);
     let pb = ProgressBar::new(progress.max_steps.into());
+    let questions = QuestionsClient::new(dinstaller_lib::connection().await.unwrap()).await.unwrap();
+    let mut question_showed = false;
     loop {
+        if !questions.is_empty().await.unwrap(){
+            if !question_showed {
+                eprintln!("There is question to answer.");
```

Comment:
I would add something like "Please, check the graphical user interface.". Well, not sure about the wording...

Reply:
I hope we can soon replace it with asking question and allow user to answer it directly from cmdline